daml-lf/archive/src/main/scala/com/digitalasset/daml/lf/archive/DecodeV1.scala

```diff
@@ -983,6 +983,12 @@ private[archive] class DecodeV1(minor: LV.Minor) extends Decode.OfPackage[PLF.Pa
           assertSince(LV.Features.typeRep, "Expr.type_rep")
           ETypeRep(decodeType(lfExpr.getTypeRep))
 
+        case PLF.Expr.SumCase.MAKE_ANY_EXCEPTION =>
+          throw ParseError("Expr.MAKE_ANY_EXCEPTION") // TODO #8020
```

Comment:
please use the full url of the issue. 
// FIXME: https://github.com/digital-asset/daml/issues/7788